lib/widgets/modular_widgets/transfer_widgets/receive/receive_large.dart

```diff
@@ -180,9 +187,10 @@ class _ReceiveLargeCardState extends State<ReceiveLargeCard> {
   }
 
   String _getQrString() {
-    return '${_selectedToken.symbol.toLowerCase()}:$_selectedSelfAddress?zts='
-        '${_selectedToken.tokenStandard}'
+    var qrData = '${_selectedToken.symbol.toLowerCase()}:'
+        '$_selectedSelfAddress?zts=${_selectedToken.tokenStandard}'
         '&amount=${_getAmount()}';
+    return qrData.substring(0, min(qrData.length, 122));
```

Comment:
Won't this cause the Qr code to not match what the user inputs and potentially cause the sender to send the wrong amount if they use the Qr code? Though it's probably a very rare edge case.

Reply:
We could do something like this to just disable the Qr code if it goes over the limit (starting from line 103):
```
_getQrString().length <= _maxQrCodeInputLength
                    ? ReceiveQrImage(
                        data: _getQrString(),
                        size: 150.0,
                        tokenStandard: _selectedToken.tokenStandard,
                        context: context,
                      )
                    : Container(
                        width: 170.0,
                        height: 170.0,
                        decoration: BoxDecoration(
                          border:
                              Border.all(color: AppColors.lightHintTextColor),
                          borderRadius: const BorderRadius.all(
                            Radius.circular(15.0),
                          ),
                        ),
                        child: const Center(
                          child: Padding(
                            padding: EdgeInsets.all(5.0),
                            child: Text(
                              'Input is too long for QR code.',
                              textAlign: TextAlign.center,
                              style: TextStyle(
                                  color: AppColors.lightHintTextColor),
                            ),
                          ),
                        ),
                      ),

Reply:
> Won't this cause the Qr code to not match what the user inputs and potentially cause the sender to send the wrong amount if they use the Qr code? Though it's probably a very rare edge case.
Yes it would and I agree better to disable the Qr like you're suggesting. I'll test and push the adjustment.